README.md

```diff
@@ -1 +1,44 @@
-# Project1
\ No newline at end of file
+# Project1
+Hello Microverse
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. The goal here is to master all of the tools and best practices I learned . All about gitflow , linters.
+
+set up a new repository and prepare it for development using best practices.
+Built With
+
+    HTML
+    CSS
+    HTML & CSS Linter
+
+Getting Started
+
+Please get your browser update. To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal:
+git clone in your terminal
+cd hello-microverse
+start index.html
+Authors
+
+👤 Author
+
+    GitHub: @VSeray
+    Twitter: @SerayVana
+    LinkedIn: LinkedIn
```

Comment:
- [ ] Please kindly consider adding links to your github, Twitter and LinkedIn pages here is how you can add links in `readme` file: 

`- GitHub: [github](https://github.com/VSeray)`

This way your readme will be more professional 👍🏻